wordpress-mobile/WordPress-iOS
# Support new WordPressAuthenticator configuration and API methods

This task allows WordPress-iOS to support the new configuration and API method changes found in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/10

## To test login changes
1. Clean project
2. `pod install`
3. Run project

**Expected Behavior**
There should be no changes to the Login UI in WordPress-iOS. If there are, then a style definition was incorrectly applied or incorrectly defined in `WordPressAuthenticator-iOS`.

## To test Aztec FancyAlert prompt changes
1.  Revert the `HACK2` commit
2. `pod install`
3. Run project
4. Go to a test blog or throwaway blog account
5. Attempt to write a post
6. A "confirm email" prompt should appear

**Expected Behavior**
The confirm email prompt buttons should remain styled the same as they were before. 

Before changes:


After changes:
![after copy](https://user-images.githubusercontent.com/1062444/42397701-7446b886-812c-11e8-952f-7a1607e7078e.png)
